x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -60,30 +57,27 @@ const ItemBar = styled<IBarStyleProps, any>('div')`
   background-color: ${props => props.color};
 `;
 
-const ItemLabel = styled.div`
-  white-space: nowrap;
+const ItemText = styled.span`
   position: absolute;
   right: 0;
-  width: auto;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  display: inline-block;
-  text-align: left;
-  margin: 0;
+  display: flex;
+  align-items: center;
+  height: ${px(units.plus)};
+
+  & > * {
```

Comment:
Mind adding this above this line: `/* all direct descendants of this element */` or similar?